tailwind.config.js

```diff
@@ -0,0 +1,123 @@
+module.exports = {
+  theme: {
+    borderColor: theme => ({
+      1: theme("colors.border.1"),
+      2: theme("colors.border.2"),
+      3: theme("colors.border.3"),
+      4: theme("colors.border.4"),
+      5: theme("colors.border.5"),
+    }),
```

Comment:
It looks like there is a naming conflict between `borderColor` and `borderWidth`:

```
`@apply` cannot be used with .border-1 because .border-1 is included in multiple rulesets.
```

This is because the border-width utility takes the form `border-#` while border-color takes the form `border-$colorkey`.

@apply `border` also seems to bomb - this may be related.